ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ScannerV4IntegrationForm.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { ReactElement } from 'react';
+import { TextInput, PageSection, Form, ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import * as yup from 'yup';
+
+import { ScannerV4ImageIntegration } from 'types/imageIntegration.proto';
+
+import FormMessage from 'Components/PatternFly/FormMessage';
+import FormTestButton from 'Components/PatternFly/FormTestButton';
+import FormSaveButton from 'Components/PatternFly/FormSaveButton';
+import FormCancelButton from 'Components/PatternFly/FormCancelButton';
+import useIntegrationForm from '../useIntegrationForm';
+import { IntegrationFormProps } from '../integrationFormTypes';
+
+import IntegrationFormActions from '../IntegrationFormActions';
+import FormLabelGroup from '../FormLabelGroup';
```

Comment:
Good candidate for upcoming tech-debt sprint cleanup:

We made a copy of this component in `ui/apps/platform/src/Components/PatternFly/FormLabelGroup.tsx`, but we must not have switched every usage of that yet.

You could go ahead and change it just here, if you have to push another commit for other reasons, or wait until the tech debt sprint to catch them all.

Reply:
oh good looks. it might be worth addressing altogether for the tech debt sprint and get this in if there's nothing else

Reply:
ended up resolving it. thanks!